test/e2e/volume_create_test.go

```diff
@@ -60,6 +60,28 @@ var _ = Describe("Podman volume create", func() {
 		Expect(len(check.OutputToStringArray())).To(Equal(1))
 	})
 
+	It("podman create and export volume", func() {
+		found := false
+		exportedVolume := "/tmp/myvol.tar"
+
+		session := podmanTest.Podman([]string{"volume", "create", "myvol"})
+		session.WaitWithDefaultTimeout()
+		volName := session.OutputToString()
+		Expect(session).Should(Exit(0))
+
```

Comment:
Please run a container and write something to the volume and then check the generated tar for that content.

Reply:
@Luap99 Please check again added equivalent test in latest commit.

Reply:
@Luap99 so i have modified it, not checking the tar but content of generated tar from stdout. PTAL